README.md

```diff
@@ -1,38 +1,105 @@
+- Yarn v1.x ([see installation instructions](https://classic.yarnpkg.com/en/docs/install))
+
+To get the app running locally:
+
+1.  Clone this repository and `cd` into it
+
+2.  Install the dependencies
+
+        yarn
+
+3.  Make a copy of `.env.sample` called `.env.local` and complete as follows:
+
+| Name                            | Description                                                                                   |
+| ------------------------------- | --------------------------------------------------------------------------------------------- |
+| ENV                             | The name of your environment, `local` is a good value here                                    |
+| ENDPOINT_API                    | The URL for the HTTP API                                                                      |
+| AWS_KEY                         | The API key for calling the HTTP API (defined in `ENDPOINT_API`)                              |
+| GSSO_URL                        | The URL for the authentication service at Hackney                                             |
+| GSSO_TOKEN_NAME                 | The local storage key name where the authentication JWT should be stored                      |
+| HACKNEY_JWT_SECRET              | The shared secret used for validating the JWT                                                 |
+| AUTHORISED_ADMIN_GROUP          | The name of the Admins Google Group                                                           |
+| AUTHORISED_ADULT_GROUP          | The name of the Adult (ASC) Google Group                                                      |
+| AUTHORISED_CHILD_GROUP          | The name of the Childrens (CFS) Google Group                                                  |
+| AUTHORISED_ALLOCATORS_GROUP     | The name of the Allocators Google Group                                                       |
+| AUTHORISED_UNRESTRICTED_GROUP   | The name of the Unrestricted Google Group                                                     |
+| AUTHORISED_DEV_GROUP            | The name of the Developers Google Group                                                       |
+| POSTCODE_LOOKUP_URL             | The URL for the postcode lookup service (including `/api/v1/addresses?postcode=` at the end!) |
+| POSTCODE_LOOKUP_APIKEY          | The API key for the postcode lookup service (defined in `POSTCODE_LOOKUP_URL`)                |
+| REDIRECT_URL                    | The URL to redirect to post-authentication (`dev.hackney.gov.uk:3000` for local development)  |
+| NEXT_PUBLIC_GOOGLE_ANALYTICS_ID | The ID for Google Analytics data to be reported to                                            |
+| NEXT_PUBLIC_FEEDBACK_LINK       | The Google Form URL that powers the feedback link                                             |
+| STORYBOOK_API_PROXY             | ???                                                                                           |
```

Comment:
Does anyone know if this is actually used? I'll remove if not

Reply:
What do you think of moving the descriptions / comments into the `env.sample` file?
We can still point to the file , the comments remain close to the usage and would be easier to maintain

Reply:
Yeah really nice idea @neilkidd – done in https://github.com/LBHackney-IT/lbh-social-care-frontend/pull/454/commits/bd94500ddb8d2f1492fd98c083905989377c291d. Let me know what you think?

Reply:
🏆 🥇